content/scalability/docs/control-plane.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes Control Plane
+
+The Kubernetes control plane is the Kubernetes components such as the Kubernetes API Server, Kubernetes Controller Manager, and other components that are required for Kubernetes to funciton. Scalability limits of these components are different depending on what you’re running in the cluster, but the areas with the biggest impact to scale include the Kubernetes version, utilization, and individual Node scaling.
+
+## Use EKS 1.24 or above
+
+EKS 1.24 introduced a number of changes and switches the default container runtime to [containerd](https://containerd.io/) instead of docker. Containerd helps clusters scale by increasing individual node performance by limiting container runtime features to closely align with Kubernetes’ needs. Containerd is available in every supported version of EKS and if you would like to switch to containerd in versions prior to 1.24 please read the documentation about [Dockershim deprecation](https://docs.aws.amazon.com/eks/latest/userguide/dockershim-deprecation.html).
+
+## Limit workload and node bursting
+
+The EKS control plane will automatically scale as your cluster grows, but there are limits on how fast it will scale. When you first create an EKS cluster the Control Plane will not immediately be able to scale to hundreds of nodes or thousands of pods. To avoid reaching API limits on the control plane you should limit scaling spikes that increase cluster size by double digit percentages at a time (e.g. 1000 nodes to 1100 nodes or 4000 to 4500 pods at once). To read more about how EKS has made scaling improvements see [this blog post](https://aws.amazon.com/blogs/containers/amazon-eks-control-plane-auto-scaling-enhancements-improve-speed-by-4x/).
+
+Scaling large applications requires infrastructure to adapt to become fully ready (e.g. warming load balancers). To control the speed of scaling make sure you are scaling based on the right metrics for your application. CPU and memory scaling may not accurately predict your application constraints and using custom metrics (e.g. requests per second) in Kubernetes Horizontal Pod Autoscaler](hPA) may be a more accurate scaling option.
+
+To use a custom metric see the examples in the [Kubernetes documentation](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics).
+
+## Scale nodes and pods down safely using Pod Disruption Budgets or Karpenter TTL Settings
+
+Use Karpenter’s [time to live (TTL)](https://aws.github.io/aws-eks-best-practices/karpenter/#use-timers-ttl-to-automatically-delete-nodes-from-the-cluster) settings to replace instances after they’ve been running for a specified amount of time. Self managed node groups can use the `max-instance-lifetime` setting to cycle nodes automatically. Managed node groups do not currently have this feature but you can track the request [here on GitHub](https://github.com/aws/containers-roadmap/issues/1190).
```

Comment:
Can you make the recommendations headings? This way they will appear in the right hand nav.

Reply:
How does recycling your nodes impact scalability? I agree it is a good practice, but I'm having trouble seeing the connection.